net.kuribo64.melonDS.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be upstreamed if possible as it's not a flatpak/flathub specific file.

Reply:
Will look into getting that done in a few days.